neo/Prompt/Commands/tests/test_send_commands.py

```diff
@@ -223,6 +223,28 @@ def test_insufficient_funds(self):
             self.assertFalse(res)
             self.assertIn("Insufficient funds", mock_print.getvalue())
 
+    def test_transaction_size_1(self):
+        with patch('sys.stdout', new=StringIO()) as mock_print:
+            with patch('neo.Core.TX.Transaction.Transaction.Size', return_value=1026):  # returns a size of 1026
+                PromptData.Wallet = self.GetWallet1(recreate=True)
+                args = ['send', 'gas', self.watch_addr_str, '5']
+
+                res = Wallet.CommandWallet().execute(args)
+
+                self.assertFalse(res)
+                self.assertIn('Transaction cancelled. The tx size (1026) exceeds the max free tx size (1024).\nA network fee of 0.001 GAS is required.', mock_print.getvalue())
```

Comment:
it would be nice if we could add the same comment as you did for deploy
```
# notice the required fee is equal to the low priority threshold
```
I found that those were really useful in quickly understanding the reasoning behind the required value.

Reply:
Okay will do!